runtime-complexity/memoization.md

```diff
@@ -1,212 +1,120 @@
 # Memoization
 
-  ### Projected Time
+### Projected Time
+ About 90 minutes
 
-  90 minutes
+### Prerequisites
+- [Runtime Complexity](https://github.com/Techtonica/curriculum/blob/master/runtime-complexity/runtime-complexity.md)
 
-  ### Prerequisites
-
-  Here are links to lessons that should be completed before this lesson:
-
-  -  [Runtime Complexity](https://github.com/Techtonica/curriculum/blob/master/runtime-complexity/runtime-complexity.md)
-
-  ### Motivation
-
-  Learning memoization will help improve the runtime of your code which is a win for you and a win for your work.
-The simple goal of memoization is to not waste time having your code do something is has already done. You can simply re-use the part of the code that was already run!
-
-  ### Objectives
-
-  **Participants will be able to:**
+### Motivation
+- Learning memoization will help improve the runtime of your code which is a win for you and a win for your work.
+- The simple goal of memoization is to not waste time having your code do something it has already done. You can simply re-use the part of the code that was already run!
 
+### Objectives
+**Participants will be able to:**
 - Understand memoization
-
 - Recognize the benefit in runtime complexity when using memoization
-
 - Write code that uses memoization
+- Explain the differences in runtime with memoization, tabulation, and regular recursion
 
-- Explain the differences in runtime with memoization, tabulation and regular recursion
-
- ### Specific Things To Teach
-
- - What is memoization
-
+### Specific Things To Teach
+- What is memoization
 - How does memoization improve Runtime Complexity
-
 - How does memoization compare to recursion
 
 ### Implementation of  memoization
 Writing memoization into code can be very simple:-
-
 - create a hash table
-
 - add any results in the hash table
-
 - when new results are derived check to see if those results are in the hash table;
-
 - if so, call on the results from the hash table to save time, if not, add the new results to the hash table
 
 ### Materials
-
-  -  [Memoization Slideshow](https://docs.google.com/presentation/d/1BipDMgjZd3u-QsrPNCljH-Wv2l3tYRAUz8LWnxzt4s8/edit#slide=id.p)
-
+-  [Memoization Slideshow](https://docs.google.com/presentation/d/1BipDMgjZd3u-QsrPNCljH-Wv2l3tYRAUz8LWnxzt4s8/edit#slide=id.p)
```

Comment:
@alodahl I don't have edit access for this slide. So I can't review it.

Reply:
@vsharma20v Make sure that the slideshow follows all the formatting guidelines mentioned in #600